README.md

```diff
@@ -27,10 +34,299 @@ To build all apps and packages, run the following command:
 pnpm run build
 ```
 
+You can also build a specific app or package by running this command:
+
+```bash
+pnpm dlx turbo run build --filter=checkout
+# or
+pnpm run build:checkout
+```
+
+In this example, we'll only build `apps/checkout`
+
 ### Develop
 
-To develop all apps and packages, run the following command:
+Create a tunnel for `checkout-app`:
+
+```bash
+cd apps/checkout-app && npx saleor app tunnel 3000
+```
+
+> Note: the process needs to be running in the background
+
+Before you start the server, you need to change default environment variables. Create `.env.local` file in each app:
+- [`apps/checkout-app`](./apps/checkout-app/README.md#env-variables)
+- [`apps/checkout`](./apps/checkout/README.md#local-development)
+
+To run the development server for all the apps, use the following command:
 
 ```
 pnpm run dev
 ```
+
+You can also run only a specific app by running this command:
+
+```bash
+pnpm dlx turbo run dev --filter=checkout
+# or
+cd apps/checkout && pnpm dev
+```
+
+## Deployment
+
+### Env variables
+
+Change environment variables inside `.env` file:
+  - `SALEOR_API_URL` — GraphQL endpoint of your Saleor
+
+    Example:
+
+    ```
+    https://my-env.eu.saleor.cloud/graphql/
+    ```
+    > To run sandbox Saleor environment in [Saleor Cloud](https://cloud.saleor.io/) use this command:
+    > ```bash
+    > npx saleor project create && npx saleor environment create
+    > ```
+    > You can also run Saleor locally. See [Saleor docs](https://docs.saleor.io/docs/3.x/developer/installation) for more instructions
+  - `CHECKOUT_API_URL` — API endpoint of deployed Checkout App
+
+    Example:
+    ```
+    https://saleor-checkout-app.vercel.app/api
+    ```
+    > See [guide below](#checkout-app) on how to deploy the Checkout App
+
+There are more environment variables available in each app. Go to their README's to learn more
+
+### Vercel
+
+The repo needs to be hosted on GitHub or some other git repository. Before you start, fork the repo to your account or organization.
+
+- Authenticate the Turborepo CLI with your Vercel account
+
+```
+pnpm dlx turbo login
```

Comment:
Is there a shorter version like with the above ones?

Reply:
We could use `npx turbo login` but this way we would use another package manager. I could also write just `turbo login` but would need to assume that the `turbo` package is installed globally by the user

Reply:
OK, thanks :slightly_smiling_face: